ta/pkcs11/src/processing.h

```diff
@@ -54,4 +56,10 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 
 enum pkcs11_rc tee_init_ctr_operation(struct active_processing *processing,
 				      void *proc_params, size_t params_size);
+
+enum pkcs11_rc tee_init_derive_symm(struct active_processing *processing,
```

Comment:
do we really need to export this function? It is only used in processing_symm.c.

Reply:
Made it a static function